server/implementation/pom.xml

```diff
@@ -42,6 +42,11 @@
             <artifactId>jakarta.json-api</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>jakarta.validation</groupId>
+            <artifactId>jakarta.validation-api</artifactId>
+            <version>2.0.2</version><!-- TODO should we manage this version? -->
+        </dependency>
```

Comment:
Can this not be provided ?

Reply:
Yes, of course... I just forgot. Will fix it.